### PR TITLE
Update methods for feeding data

### PR DESCRIPTION
**Pull request by Cisphyx**

### CODE REVIEW

File: synapse/lib/layer.py [hunk 1]
```diff
@@ -105,15 +105,39 @@ class LayerApi(s_cell.CellApi):
...
         '''
         Scan the full layer and yield artificial nodeedit sets.
         '''
+
         await self._reqUserAllowed(self.liftperm)
         async for item in self.layr.iterLayerNodeEdits():
             yield item
 
+    async def storNodeEdits(self, nodeedits):
+
```
Comment (mikemoritz) [inline synapse/lib/layer.py]: For splice -> nodeedit migration case I do have meta data, but we would be re-generating it here.  What do you think about allowing meta to be passed in?

Comment (Cisphyx) [reply synapse/lib/layer.py]: Makes sense, I'll update it.

File: synapse/cortex.py [hunk 2]
```diff
@@ -401,18 +401,25 @@ def getCoreInfo(self):
         '''
         return await self.cell.getFeedFuncs()
 
-    async def addFeedData(self, name, items, seqn=None):
+    async def addFeedData(self, name, items, *, viewiden=None):
 
-        wlyr = self.cell.view.layers[0]
+        if viewiden is not None:
```
Comment (jnwatson) [inline synapse/cortex.py]: Slightly redundant, since cell.getView accepts None and returns the right view.

Comment (Cisphyx) [reply synapse/cortex.py]: Removed.

**Pull request merged**